lerobot/common/policies/policy_protocol.py

```diff
@@ -49,20 +49,23 @@ def reset(self):
 
         Does things like clearing caches.
         """
+        ...
```

Comment:
I'm following PEP 544 for protocol definition/style

- https://peps.python.org/pep-0544/#defining-a-protocol
- https://typing.readthedocs.io/en/latest/spec/protocol.html#protocols

Reply:
Hmm. I think we should make this change if we decide we want to follow this style pointer. But we haven't made that determination, and on a higher level we currently want to avoid being that specific about style.

For instance, if this were amidst a bunch of other changes (like a whole new class) I'd probs leave it as is without comment. But here, we'd be changing the status quo without any overarching reason to do so. Mind if we leave it out?